plugins/woocommerce/tests/e2e-pw/tests/merchant/create-simple-product.spec.js

```diff
@@ -105,6 +114,7 @@ test.describe( 'Add New Simple Product Page', () => {
 		await page.goto( 'wp-admin/post-new.php?post_type=product' );
 		await page.fill( '#title', nonVirtualProductName );
 		await page.fill( '#_regular_price', productPrice );
+		await expect( page.locator( '#publish:not(.disabled)' ) ).toBeVisible();
```

Comment:
There are times when the Publish button wasn't enabled yet by the time Playwright gets to this part of the test. This line ensures that it's enabled before performing the click.